packages/itwinui-css/src/global.scss

```diff
@@ -27,6 +21,17 @@
   }
 }
 
+// reset margin only for <body> element.
+:where(body.iui-root) {
+  margin: 0;
+}
+
+// auto apply background only if <body> is used as root.
+// for other elements, user will have to opt in by adding .iui-root-bg
+:where(body.iui-root, .iui-root.iui-root-bg) {
```

Comment:
nit: One small consideration could be whether to name it `.iui-root-bg` or `.iui-root-background`. Because I think all public class names/css variables have "background" and not "bg".